focus-android/app/src/main/res/values-sw600dp/dimens.xml

```diff
@@ -4,4 +4,6 @@
    - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
 <resources>
     <dimen name="firstrun_container_width">320dp</dimen>
+    <dimen name="tablet_bottom_sheet_width">480dp</dimen>
+    <dimen name="tablet_bottom_sheet_peekheight">320dp</dimen>
```

Comment:
Similarly, should `tablet_bottom_sheet_width` here be 600?

Reply:
These are the original values, I do not think we should change them.

Reply:
See the deleted file values and also [this](https://github.com/mozilla-mobile/firefox-android/blob/06b398cf5fd7973f825dfe28b316a3d870913ade/focus-android/app/src/main/java/org/mozilla/focus/open/OpenWithFragment.java#L99) comment.